app/models/user.rb

```diff
@@ -1,10 +1,13 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] One of the requirements for this project milestone is to implement the confirmation of email for a user. Kindly implement this feature. 👍 😎 

![image](https://user-images.githubusercontent.com/77106962/226987995-090d066a-c447-4050-a7e4-482c06339756.png)

Reply:
@odaialazzeh As is requested by requirements and previous reviewer. Send the confirmation of email.
- I'm seeing on the console that's working, but I'm not receiving the confirmation email in my mailbox. Kindly fix it! 💯 
- It would be great to make the forgot password functionality send the corresponding email too! 🥇

Reply:
@BregornOriginal you don't receive the confirmation email in the mailbox because the application is running locally and configured to send email notifications to the console instead of sending them to the email address.

![email](https://user-images.githubusercontent.com/111691279/227048624-4e6e2d5a-f2d4-4992-8a5e-347da46ea13e.png)
so when you click on the link the email will be confirmed.